discovery-infra/start_discovery.py

```diff
@@ -604,11 +610,13 @@ def main():
     )
     parser.add_argument(
         "--day2-cluster",
-        type=distutils.util.strtobool,
-        nargs='?',
-        const=False,
-        default=False,
-        help="day 2 cluster creation"
+        help="day2 cluster",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--day1-cluster",
+        help="day1 cluster",
```

Comment:
i don't understand this flag.

Reply:
why else was not enough?

Reply:
we need to be able to run 3 scenarios with the same target (deploy_nodes):
1) run day1 only
2) run day2 only
3) run day one and then day2
so, day1-cluster means that we need to run day1 installation